frontend/javascripts/oxalis/view/layouting/tracing_layout_view.tsx

```diff
@@ -401,7 +374,6 @@ function mapStateToProps(state: OxalisState) {
     isUpdateTracingAllowed: state.tracing.restrictions.allowUpdate,
     showVersionRestore: state.uiInformation.showVersionRestore,
     storedLayouts: state.uiInformation.storedLayouts,
-    isDatasetOnScratchVolume: state.dataset.dataStore.isScratch,
```

Comment:
Maybe also remove the property from the frontend as this is the onliest place where it is actively used?